### PR TITLE
Switch to @link in the supergraph

### PR DESCRIPTION
**Pull request by pcmanus**
Generates supergraph with `@link` directives instead of `@core` ones.

### CODE REVIEW

File: composition-js/src/merging/merge.ts [hunk 1]
```diff
@@ -295,15 +295,15 @@ class Merger {
 
     for (const type of this.merged.types()) {
       // We've already merged unions above
-      if (type.kind === 'UnionType' || joinSpec.isSpecType(type)) {
+      if (type.kind === 'UnionType' || [linkSpec, joinSpec].some((s) => s.isSpecType(type))) {
```
Comment (benweatherman) [inline composition-js/src/merging/merge.ts:298]: **style preference** To mine eyes, this is still a little easier to understand in a hurry

```suggestion
      if (type.kind === 'UnionType' || linkSpec.isSpecType(type) || joinSpec.isSpecType(type) {
```

**Review by benweatherman — APPROVED** [review]
🐐 left an unimportant comment, do with it as you wish

File: composition-js/src/merging/merge.ts (hunk 1 above)
Comment (pcmanus) [reply composition-js/src/merging/merge.ts]: Agreed, it reads a tad better. Thanks.

Comment (benweatherman) [reply composition-js/src/merging/merge.ts]: omg except I'm the worst and forgot the trailing `)`

😞

Comment (pcmanus) [reply composition-js/src/merging/merge.ts]: Hahah. That's ok, I'll fix.

Comment (benweatherman) [reply composition-js/src/merging/merge.ts]: You didn't _have_ to, you just _like_ to 😸

**Pull request merged**